daemon/src/setup_contract.rs

```diff
@@ -383,13 +395,15 @@ impl RolloverParams {
         leverage: Leverage,
         refund_timelock: u32,
         fee_rate: u32,
```

Comment:
Rename this to `miner_fee_rate` to distinguish it better?

Reply:
Or `blockchain_fee_rate`?

Reply:
`tx_fee_rate`?

Reply:
renamed to `tx_fee_rate`